content/projects/gsoc/2020/projects/winsw-yaml-configs.adoc

```diff
@@ -21,25 +21,22 @@ links:
   idea: /projects/gsoc/2020/project-ideas/winsw-yaml-config-support
 ---
 
-// TODO: add details here, cannot embed draft due to wrong format
-
-On Windows machines, Jenkins master and agents can be installed as Windows Services in order to get better robustness and manageability within the system.
+On Windows machines, Jenkins master and agents can be installed as Windows Services to get better robustness and manageability within the system.
 This is a functionality bundled into the Jenkins core directly.
 When installed as a service, Jenkins uses the Windows Service Wrapper executable (.NET, written in C#) which is being configured by XML config files.
 Currently, there are only a few configuration checks there (no XML Schema, limited validation, etc.),
 and it’s often that the service wrapper is misconfigured by Jenkins users.
 
-In this project we propose to update Windows Service Wrapper to support YAML files as configuration inputs and to introduce better configuration validation during the service installation and startup.
-Usage of YAML should simplify configuration management in Jenkins, especially when automated tools are used.
+In this project, we propose to update Windows Service Wrapper to support YAML files as configuration inputs and to introduce better configuration validation during the service installation and startup.
+The usage of YAML should simplify configuration management in Jenkins, especially when automated tools are used.
 
-=== Deliverables
+== Deliverables
 
-* Service Descriptor : YAML Support
+* Service Descriptor: YAML Support
 * Validations for Configs
 * Schema validation for YAML (JSON)
 * Unit Test for Validations
 * Project Guideline Documentation
-* Informative logger
 * Sample YAML config files
 ** Sample-alloption.yaml
```

Comment:
AFAICT there are only *sample-allOptions.xml* and *sample-allOption.yml*. And you missed an *s* in the latter.